tsconfig.json

```diff
@@ -3,12 +3,16 @@
         "lib": ["es2015", "es2016.array.include", "es2017.object"],
         "noEmit": true,
         "target": "esnext",
+        "module": "commonjs",
```

Comment:
Does this do anything? We don't exactly have code generation.

Reply:
No, that can be removed.